frame/evm/src/runner/mod.rs

```diff
@@ -77,7 +77,7 @@ pub trait Runner<T: Config> {
 		is_transactional: bool,
 		validate: bool,
 		weight_limit: Option<Weight>,
-		proof_size_base_cost: Option<u64>,
```

Comment:
Is preferable to keep this as an `Option` then doing `proof_size_pre_execution.unwrap_or_default()`

Reply:
but the value doesn't make sense as None, since none will be translated to zero anyway. So I chose to have it as u64 from the beginning